uint8 color  
uint8 where   # Colocar o n√∫mero da figura
uint8 AZUL=13
uint8 VERDE=12
uint8 VERMELHO=14
uint8 DESCONHECIDO=255
---

